src/utils/tokens.ts

```diff
@@ -1,7 +1,7 @@
 import { Address, Client, Hex, PublicClient, getContract } from 'viem';
 import { vaultV2Abi, vaultExtensionAbi_V3 } from '../abi';
-import { VAULT_V2 } from './constantsV2';
-import { VAULT_V3 } from './constantsV3';
+import { balancerV2Contracts } from './balancerV2Contracts';
+import { balancerV3Contracts } from './balancerV3Contracts';
```

Comment:
Example of what i mean above ^

Reply:
I like the approach importing from `balancerV3Contracts` more than importing `VAULT_V3` which imports from `balancerV3Contracts`

Reply:
Same here 👍
Importing directly from balancerContracts files seem to make more sense.